docs/VnVPlan/VnVPlan.tex

```diff
@@ -85,60 +85,52 @@ \section{Symbols, Abbreviations, and Acronyms}
+  
+  \subsection{2.2 Objectives}
+  
+  The primary objectives of the Verification and Validation (V\&V) plan for Orbit Watch are:
+  
+  \begin{itemize}
+      \item \textbf{Ensure High Data Accuracy}: Build confidence in the software's correctness by verifying that the system achieves high classification accuracy for objects reported in the images. This is crucial for extracting useful information and providing value to stakeholders.
+      \item \textbf{Validate System Reliability and Accessibility}: Confirm that the system is reliable with minimal downtime and is accessible remotely for both purchasers and labelers, ensuring flexibility and efficiency in usage.
+  \end{itemize}
+  
+  The following objectives are out of scope for this V\&V plan:
+  
+  \begin{itemize}
+      \item \textbf{Usability Testing (Ease of Use)}: Due to limited resources, we will not conduct extensive usability testing to verify the ease of use of the platform. We assume that the user interface adheres to standard design principles that facilitate user interaction with minimal training.
+      \item \textbf{Verification of External Components}: We will not verify external libraries or services used for image acquisition, payment processing, or AI algorithms. It is assumed that these components have been adequately verified by their respective development teams.
+      \item \textbf{Stretch Goals}: Objectives related to automatic data labeling and multi-source integration are considered stretch goals and will not be included in the current V\&V efforts due to time and resource constraints.
+  \end{itemize}
+  
+  \subsection{2.3 Challenge Level and Extras}
+  
+  This project is classified at the \textbf{advanced} challenge level due to several factors:
+  
+  \begin{itemize}
+      \item \textbf{Limited Domain Knowledge}: The team has minimal experience with satellite imagery and computer vision models, requiring additional research and learning to implement effective solutions.
+      \item \textbf{Complex Implementation}: Developing a web application from scratch involves challenges in front-end development, integrating secure payment systems, handling parallel image labeling tasks, and designing consensus algorithms for data accuracy.
+      \item \textbf{Integration of Multiple Components}: The project demands seamless integration of crowd-sourcing models, AI algorithms, payment processing, and user interfaces, increasing the complexity of development and testing.
+  \end{itemize}
+  
+  No additional extras, such as formal proofs or GenderMag personas, are included in this project as per the agreed-upon problem statement.
```

Comment:
You could still mention the extras we have anyway if time permits. They should be listed in the dev plan